Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -131,12 +279,143 @@ Note that the hash is based on a random secrect that is different for
 each run. This avoids dictionary attacks.
```

Comment:
Should we say why this is done? (To prevent that someone figures out the original value from the hash)

Reply:
done